### PR TITLE
css can be ommited

### PR DESCRIPTION
**Pull request by calebx**
@Darmody 
works?
have not try.

### CODE REVIEW

**Comment by Darmody** [comment]
Good point. We should fine a way to test this probject.

File: bin/component.js [hunk 1]
```diff
@@ -56,18 +65,28 @@ function generateStylesFile(componentPath, data) {
...
+      componentPath: componentPath,
+      componentName: componentName,
+      className: humps.decamelize(componentName, { separator: '-' })
+    };
+  }
+  else if (result.name === 'cssFileIncluded') {
+    answerData.cssFileIncluded = result.answer;
+  }
+
+  console.log(answerData);
```
Comment (Darmody) [inline bin/component.js]: unnecessary log

File: bin/component.js [hunk 2]
```diff
@@ -5,13 +5,14 @@ var mkdirp = require('mkdirp');
 var inquirer = require('inquirer');
 var chalk = require('chalk');
 var tangular = require('tangular');
-var path = require('path')
-var isFilePath = require('is-valid-path')
+var path = require('path');
+var isFilePath = require('is-valid-path');
 var humps = require('humps');
 
 tangular.settings.delimiters = ['$', '$'];
```
Comment (Darmody) [inline bin/component.js]: delimiters have changed to `$`.

File: templates/component.js [hunk 3]
```diff
@@ -1,16 +1,24 @@
 import React from 'react'
 import _ from 'ramda'
 import setDisplayName from 'recompose/setDisplayName'
+{{if cssFileIncluded}}
 import CSSModules from 'react-css-modules'
 import styles from './styles.scss'
+{{fi}}
```
Comment (Darmody) [inline templates/component.js]: Should use `$` instead of `{{` and `}}` in template file. ⬇️ ⬇️ ⬇️

File: bin/component.js (hunk 2 above)
Comment (calebx) [inline bin/component.js]: yeah, changed to `$if$`

**Comment by Darmody** [comment]
commit 需要整理一下

**Comment by calebx** [comment]
rebased

**Comment by Darmody** [comment]
LGTM

**Pull request merged**